web/components/header.tsx

```diff
@@ -28,6 +32,49 @@ export const Header: React.FC<HeaderProps> = ({ signOut, setShowSidebar }) => {
   const showWallet = process.env.REACT_APP_SHOW_CONNECT_WALLET === 'true';
   const { doOpenAuth } = useConnect();
   const name = bnsName();
+  const contractAddress = process.env.REACT_APP_CONTRACT_ADDRESS || '';
+  const stxAddress = useSTXAddress();
+  const [isVotingOpen, setisVotingOpen] = useState(false);
+
+  useEffect(() => {
+    let mounted = true;
+
+    const getData = async () => {
+      const proposals = await callReadOnlyFunction({
```

Comment:
REQ: we don't need to fetch v2 proposals since there will never be an open one there

Reply:
Yeah. I wasn't sure. I'll change this. 👍